File: HelperJobby/DTOs/Resume/WorkExperienceDTO.cs
namespace HelperJobby.DTOs.Resume;

public class WorkExperienceDTO
{
    public int WorkExperienceId { get; set; }
    public string? JobTitle { get; set; }
    public string? Company { get; set; }
    public string? Country { get; set; }
    public string? CityOrProvince { get; set; }
    public DateOnly? From { get; set; }
    public DateOnly? To { get; set; }
    public bool? CurrentlyWorkHere { get; set; }
    public string? Description { get; set; }
    public int ResumeId { get; set; }
    public ResumeDTO Resume { get; set; }
}

File: HelperJobby/client-app/src/DTOs/ResumeRelatedDTOs/WorkExperienceDTO.ts
import {ResumeDTO} from "./ResumeDTO";

export interface WorkExperienceDTO {
    workExperienceId: number;
    jobTitle?: string;
    company?: string;
    country?: string;
    cityOrProvince?: string;
    from?: Date;
    to?: Date;
    currentlyWorkHere?: boolean;
    description?: string;
    resumeId: number;
    resume: ResumeDTO;
}